src/core/connected-graph/hierarchy.ts

```diff
@@ -582,7 +582,7 @@ export class D3HierarchyGraph extends D3ConnectedGraphSVG {
         let parentMidX = rect.width / 2;
 
         let elemMidX = parentMidX - topRect.width / 2;
-        let elemMidY = self._padding + nodeContentRect.height + topRect.height / 2;
+        let elemMidY = self._padding + topRect.height / 2;
```

Comment:
The reason for this change is that on line 586, `self.transformHorizontalNodes` is being called with `node.top`.  Since `node.top` represents the collection of nodes that should be placed at the top of the parent `node`, we want those nodes to be aligned with the center of `topRect`.  With this change in place, `elemMidY` now correctly represents the y-center of `topRect`.